content/en/opentelemetry/guide/combining_otel_and_datadog_metrics.md

```diff
@@ -0,0 +1,48 @@
+
+If you want to show the average number of active nginx connections using a metrics query, there are two challenges:
+
+1. You need to know the names (and optionally metric filters) for both Datadog and OTel style naming conventions. It is often the case the query author has familiarity with one schema or the other. It would be uncommon, and burdensome, to understand the complete mapping of equivalences between integration metrics between these two standards.
+
+2. Combining two separate metric queries together using [metrics functions][1] restricts you to combining the result of two metric queries, rather than aggregate all timeseries across both metric names as a single metric.
+
+As an example, if we have two queries: `avg:nginx.net.connections`, and `avg:nginx.connections_current{state:active}`, we cannot represent the average of all contributing time series by combining these results. You cannot take an average of averages.
+
+## Datadog Makes Combining Datadog and OTel Integration Metrics Easy
+
+The metrics function `equiv_otel` is provided to address both problems illustrated above.
+
+If you have a Datadog style query for `avg:nginx.net.connections` and you want to actually examine the average of all nginx active connections regardless of whether that data is provided by a Datadog or OTel integration, simply specify your query as an argument to the `equiv_otel` function. The `equiv_otel` function will automatically convert the Datadog style query to the equivalent OTel query, and then aggregate the results of both queries together as if they were a single metric.
+
+`equiv_otel(avg:nginx.net.connections)`
+
+Upon seeing a query wrapped in `equiv_otel`, Datadog will first lookup the equivalent OTel named metric, and optionally a filter. In this case its `nginx.connections_current{state:active}`. Next all of the query aggregations, `avg` in this case, will be applied across all time series combined from these two metrics, preserving the aggregation semantics of hte original query.
+
+This function works both ways. If you have an OTel query first, but want to change it to include all equivalent Datadog integration metrics, wrap it in `equiv_otel`. In this example, it would best
+
+`equiv_otel(avg:nginx.current_connections{state:active})`
+
+Datadog will know that the corresponding time series for nginx integrations observed by Datadog is `nginx.net.connections`. It will include these time series in the `avg` aggregation.
+
+Now you don't have to worry about knowing the equivalences between integration metrics.
```

Comment:
`````suggestion
## Combining metrics with the equiv_otel function

The `equiv_otel` function automatically combines equivalent Datadog and OTel metrics in a single query. It:

- Automatically handles metric name translation.
- Properly aggregates all timeseries as a single metric.
- Works bidirectionally (Datadog to OTel or OTel to Datadog).
- Preserves query aggregation semantics.

### Converting from Datadog to OTel

To include the equivalent OTel metrics in your query, wrap your Datadog query in `equiv_otel`:

```
equiv_otel(avg:nginx.net.connections)
```
This query:
1. Identifies the equivalent OTel metric (`nginx.connections_current{state:active}`)
2. Combines timeseries from both metrics
3. Applies the aggregation (`avg`) across all datapoints

### Converting from OTel to Datadog

The same works for including Datadog metrics in an OTel query:

```
equiv_otel(avg:nginx.connections_current{state:active})
```
The function works the same way in reverse, automatically including the equivalent Datadog metric (`nginx.net.connections`).
`````
To confirm, I think `nginx.current_connections` should be `nginx.connections_current`?